appengine/appidentity/pom.xml

```diff
@@ -34,12 +34,22 @@ Copyright 2015 Google Inc. All Rights Reserved.
       <version>${appengine.target.version}</version>
     </dependency>
     <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
+    </dependency>
+    <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>servlet-api</artifactId>
       <version>2.5</version>
       <type>jar</type>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
Ok for now, but I think we should probably move our samples to GSON or something else.